src/components/dashjs-player/dashjs-player.tsx

```diff
@@ -49,9 +50,35 @@ export class DashjsPlayer {
 
   @State() streamInterval: any;
 
+  @State()
+  metrics = {
+    video: {
+      'Buffer Length': 0,
+      'Bitrate Downloading': 0,
+      'Dropped Frames': 0,
+      'Frame Rate': 0,
+      'Index': 0,
+      'Max Index': 0,
+      'Live Latency': 0,
+      'Latency': '0|0|0',
+      'Download': '0|0|0',
+      'Ratio': '0|0|0',
+    },
+    audio: {
+      'Buffer Length': 0,
+      'Bitrate Downloading': 0,
+      'Dropped Frames': 0,
+      'Max Index': 0,
+      'Latency': '0|0|0',
+      'Download': '0|0|0',
+      'Ratio': '0|0|0',
+    },
+    currentTime: '00:00',
+  };
+
   @State()
   controlbar: any;
-  // TODO: Really Bad practice! Use a better flow to get updates to statistics
+
   @Event({
     composed: true,
     bubbles: true,
```

Comment:
this shouldn't be needed anymore

Reply:
it's already removed

Reply:
I mean the event...

Reply:
playerEvent is in the main branch, not sure what you mean. I just don't want to remove some implementations that are not mine which can cause problems.